chef/cookbooks/updater/recipes/default.rb

```diff
@@ -91,6 +116,12 @@
       end # block
     end # ruby_block
 
+    ["corosync", "pacemaker"].each do |s|
```

Comment:
If we stop corosync on line 61, then we should start corosync in here as well no?